### PR TITLE
Configurable systems/event clearing

### PR DESCRIPTION
**Pull request by Aceeri**
Related issue: https://github.com/lucaspoffo/renet/issues/31

Somewhat follows bevy_rapier's example although I don't make entire stages since that seems a bit wasteful here. But this should also make it easier to add more systems without breakage in the future if needed.

### CODE REVIEW

File: bevy_renet/src/lib.rs [hunk 1]
```diff
@@ -70,11 +121,32 @@ impl RenetClientPlugin {
         }
     }
 
-    pub fn send_packets_system(mut client: ResMut<RenetClient>, mut renet_error: EventWriter<RenetError>) {
+    pub fn send_packets(mut client: ResMut<RenetClient>, mut renet_error: EventWriter<RenetError>) {
```
Comment (Shatur) [inline bevy_renet/src/lib.rs:124]: Using `_system` is a convention suggested by `@cart`. Also other systems still have `_system` suffix, so I would keep it.

**Review by Shatur — CHANGES_REQUESTED** [review]
Looks reasonable, similar to [Rapier](https://github.com/dimforge/bevy_rapier/blob/master/src/plugin/plugin.rs) but with systems instead of stages (makes sense).

File: bevy_renet/src/lib.rs [hunk 2]
```diff
@@ -56,11 +85,33 @@ impl RenetServerPlugin {
...
+                .with_system(Self::update_system.with_run_criteria(has_resource::<RenetServer>))
+                .label(RenetSystems::Update),
+            RenetSystems::SendPackets => SystemSet::new().with_system(
+                Self::send_packets
+                    .with_run_criteria(has_resource::<RenetServer>)
+                    .label(RenetSystems::SendPackets),
+            ),
+        }
+    }
 }
```
Comment (lucaspoffo) [inline bevy_renet/src/lib.rs:115]: I feel that adding systems like this adds too much noise to the code without gains, we have very few system, the labels almost map 1 to 1. I think we still could use the old way, WDYT?

Comment (Aceeri) [reply bevy_renet/src/lib.rs:115]: Ya I was just thinking it might be useful if more systems are added in the future.

Comment (lucaspoffo) [reply bevy_renet/src/lib.rs:115]: When the time comes, we can use this approach if necessary :+1: 
For now, I would just use after/before the system names and remove the `get_system`.

**Comment by lucaspoffo** [comment]
Thanks! Sorry for the delay

**Comment by Shatur** [comment]
@lucaspoffo variable `default_system_setup` is used only do setup event clearing? Maybe rename into something related to events then?

**Comment by lucaspoffo** [comment]
@Shatur yeah, I've already done this on https://github.com/lucaspoffo/renet/commit/5ac53f7e30158c0dc2e316c7caa8570c2d5763eb, when documenting it I noticed also this.

**Pull request merged**